staff/javier-lopez/cv/styles.css

```diff
@@ -0,0 +1,31 @@
+figure img{
+    border-radius: 100px;
+}
+
+body{
```

Comment:
Following the hierarchy this should be on top of the css file

Reply:
Updated!